app/router.js

```diff
@@ -7,7 +7,12 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders');
+
+  this.route('/');
+  this.route('reminders', function(){
+    this.route('index', { path: '/' });
```

Comment:
Not sure you need this either. Talk to me a little bit about what your routing situation is.